src/OSPSuite.R/Services/PKAnalysesTask.cs

```diff
@@ -10,12 +12,26 @@
 
 namespace OSPSuite.R.Services
 {
+   /// <summary>
+   /// Required because of optional DynamicParameters dependencies
+   /// </summary>
+   public class CalculatePKAnalysisArgs
```

Comment:
Shitty rClr does not work properly with Arrays... I have to encpasulate the params in an object. (I did sthg similar for createIndividuals for the same reason)